src/components/Accordion/index.tsx

```diff
@@ -18,20 +18,36 @@ export interface AccordionProps {
   /** Unique id passed to data-qa-id property of the outermost wrapper of the component. Used for automation testing  */
   dataQa?: string | null;
   noPadding?: boolean;
+  /** To set toggleAccdion become disable and not open accordion when click accodionHead  */
+  isToggleDisabled?: boolean;
 }
 
-export const Accordion = ({ title, className, children, dataQa, noPadding = false }: AccordionProps) => {
+export const Accordion = ({
+  title,
+  className,
+  children,
+  dataQa,
+  noPadding = false,
+  isToggleDisabled,
+}: AccordionProps) => {
   const [isOpen, toggleAccordion] = React.useState(false);
+
+  const onToggleAccordion = () => {
+    if (isToggleDisabled) return;
```

Comment:
how can we open the accordion if toggle is disabled?

Reply:
from the FE side i use code something like this to determined when it disabled or not
`<span
          onClick={onClickView}
          onMouseOver={() => setIsToggleDisabled(true)}
          onMouseLeave={() => setIsToggleDisabled(false)}
        >
          {translation("common.view")}
        </span>`

Reply:
ah i see. i thought the value for isToggleDisabled is static